.ruby-version.example

```diff
@@ -1 +1 @@
-1.9.3-p327
+2.1.3
```

Comment:
What do you think about dropping `.example` from this file and going with `.ruby-version`?